WORKSPACE

```diff
@@ -111,3 +111,21 @@ http_file(
     executable = True,
     sha256 = "dd8d58429a258b094b20a1435be3086ecee5d036b87c0e766309842766bc345b"
 )
+
+http_archive(
+    name = "com_google_googletest",
+    strip_prefix = "googletest-master",
+    urls = ["https://github.com/google/googletest/archive/master.zip"],
```

Comment:
If we work with `master`, the repo is no longer reproducible, since if I run it and it works, when you run it, `head` may change at that repo and it doesn't work.
How about we use a release like `com_google_protobuf` or a commit hash like here?
https://github.com/hasadna/hasadna/blob/master/WORKSPACE#L38-L42

Reply:
Worse problem: Google Test hasn't had a release since adding BUILD files. (Or, since 2016 for that matter.)

But yes, the live-at-head vs. reproducibility issue is a problem. I think the right fix is to pin the other repository by commit ID, and have an update or sync tool which updates your workspace to the latest commit ID and then you commit that workspace change to your local repository. However, Bazel recommends [new_]http_archive over [new_]git_repository. I don't really understand the motivation behind that or how they expect this to work in the larger ecosystem.

Reply:
I agree - so how about we use the http_archive with commit ID, like in:
https://github.com/hasadna/hasadna/blob/master/WORKSPACE#L38-L42

Reply:
Done.